privateid.go

```diff
@@ -0,0 +1,13 @@
+package goinsta
+
+import "fmt"
+
+const shortIDLen = 11
+
+func MediaIDFromPrivateID(code string) (string, error) {
```

Comment:
It will be great if the original function could handle private IDs too. Move these lines to the original function.